tests/system/serviceScale_test.go

```diff
@@ -18,18 +20,32 @@ func TestServiceScale(t *testing.T) {
 	defer s.Terraform.Destroy()
 
 	serviceID := s.Terraform.Output("service_id")
+	deployID := s.Terraform.Output("deploy_id")
+	scale := 3
 
-	s.Layer0.ScaleService(serviceID, 3)
+	req := models.UpdateServiceRequest{
+		DeployID: &deployID,
```

Comment:
this is testing a new deployment and we only want to test scaling here; this  should be removed.